torch/csrc/jit/ir/scope.h

```diff
@@ -143,9 +148,32 @@ struct TORCH_API InlinedCallStack : public c10::intrusive_ptr_target {
   // Return next element in the callstack list.
   c10::optional<InlinedCallStackPtr> callee() const;
 
+  // Return next element in the callstack list.
+  c10::optional<ModuleInstanceInfo> module_instance() const;
+
+  // Return next element in the callstack list.
+  SourceRange source_range() const;
+
   // Return callstack as a vector of [Function, SourceRange] pairs.
   std::vector<InlinedCallStackEntry> vec();
+
+  void setCallee(c10::optional<InlinedCallStackPtr>);
```

Comment:
`callee_` member is also of type `c10::optional<InlinedCallStackPtr>`. It could be `setCallee(InlinedCallStackPtr);` in that case optional will be constructed. This just seemed natural, but no other reason.